docs/axes/labelling.md

```diff
@@ -10,11 +10,11 @@ The scale label configuration is nested under the scale configuration in the `sc
 | -----| ---- | --------| -----------
 | `display` | `Boolean` | `false` | If true, display the axis title.
 | `labelString` | `String` | `''` | The text for the title. (i.e. "# of People" or "Response Choices").
-| `lineHeight` | `Number|String` | `1.2` | Height of an individual line of text (see [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height))
 | `fontColor` | Color | `'#666'` | Font color for scale title.
 | `fontFamily` | `String` | `"'Helvetica Neue', 'Helvetica', 'Arial', sans-serif"` | Font family for the scale title, follows CSS font-family options.
 | `fontSize` | `Number` | `12` | Font size for scale title.
 | `fontStyle` | `String` | `'normal'` | Font style for the scale title, follows CSS font-style options (i.e. normal, italic, oblique, initial, inherit).
+| `padding` | `Number` or `Object` | `0` | Padding to apply around scale labels. Only `top` and `bottom` are implemented. The `top` direction always refers to the edge of the label that is furthest aways from the chart center, i.e. for a bottom axis this `top` padding is applied at the bottom of the label.
```

Comment:
I understand. This is however the only way to set padding as global default without having different default per axis position since the bottom axis basically has reversed orientation.Can still be overridden on per-axis basis by user.